content/docs/reactivesearch/react-searchbox/searchbase.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'SearchBase API Reference'
+meta_title: 'API Reference for SearchBase'
+meta_description: 'SearchBase is a lightweight and platform agnostic search library for Elasticsearch.'
+keywords:
+    - apireference
+    - searchbase
+    - elasticsearch
+    - search library
+sidebar: 'docs'
+nestedSidebar: 'react-searchbox-reactivesearch'
+---
+
+## SearchBase
+
+It is a container component that wraps all the ReactiveSearch components together. It binds the backend app (data source) with the UI view components
+(elements wrapped within ReactiveBase), allowing a UI component to be reactively updated every time there is a change in the data source or in other UI components.
+
+### Props
+
+#### To configure the Appbase environments
+
+-   **index** `string` [required]
```

Comment:
@nikhil-varma Refer to Searchbase docs for the prop defintions.

Environment should be without (s) suffix.